trin-execution/metrics/prometheus/prometheus.yml

```diff
@@ -12,6 +12,8 @@ scrape_configs:
   # Node exporter metrics
   - job_name: "node"
     static_configs:
-      - targets: ["host.docker.internal:9091"]
+      - targets: 
+        - host.docker.internal:9091
+        - localhost:9091
```

Comment:
not sure about this one either (but it might be ok, since it has both)

Reply:
This is so both MacOS works and Linux